src/backend/distributed/deparser/deparse.c

```diff
@@ -15,216 +15,619 @@
 
 #include "postgres.h"
 
+#include "distributed/commands.h"
 #include "distributed/deparser.h"
 
-static char * DeparseDropStmt(DropStmt *stmt);
-static char * DeparseAlterTableStmt(AlterTableStmt *stmt);
-static char * DeparseRenameStmt(RenameStmt *stmt);
 static char * DeparseRenameAttributeStmt(RenameStmt *stmt);
-static char * DeparseAlterObjectSchemaStmt(AlterObjectSchemaStmt *stmt);
-static char * DeparseAlterOwnerStmt(AlterOwnerStmt *stmt);
-static char * DeparseAlterObjectDependsStmt(AlterObjectDependsStmt *stmt);
+static List * PlanRenameAttributeStmt(RenameStmt *stmt, const char *queryString);
+
+static DistributeObjectOps Aggregate_AlterObjectSchema = {
+	.deparse = (void *) DeparseAlterFunctionSchemaStmt,
```

Comment:
Unfortunately this doesn't improve our compile time guarantees, since C doesn't mind casting a 3 paramter function pointer to a 2 paramter function pointer